src/Symfony/Component/Intl/Util/GzipStreamWrapper.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl\Util;
+
+/**
+ * @internal
+ */
+class GzipStreamWrapper
+{
+    /** @var resource|null */
+    public $context;
```

Comment:
I don't see any usage of this property in this class, can it be removed?

Reply:
This property is required by the engine when implementing stream wrappers.